src/test/java/com/example/web/CarControllerRestTests.java

```diff
@@ -48,7 +48,8 @@
     @Autowired
     private TestRestTemplate restTemplate;
 
-    @MockBean
+    // As spring will configure the full context, no need to mock this bean
+    //@MockBean
```

Comment:
This test is intended to show Mockito usage with Spring so `@MockBean` is present on purpose.

Reply:
Got you.